.travis.yml

```diff
@@ -0,0 +1,20 @@
+langauge: ruby
+
+rvm:
+  - '2.2.2'
+
+services:
+  - mysql
+
+install:
+  - bundle install
+
+env:
+  - MYSQL_DATABASE='test_db' \
+    MYSQL_PASSWORD='' \
+    MYSQL_ROOT_PASSWORD='' \
+    MYSQL_TESTING_HOST='localhost' \
+    MYSQL_USER='root'
+
+script:
+  - rake spec
```

Comment:
Lets add 
`rake migrate`
`rake rubocop` (this is going to be great)

we should probably add a script somewhere to do some one off things like import data and stuff